swagger-confluence-client.yaml

```diff
@@ -44,6 +44,33 @@ paths:
           description: "Returned if there is no space with the given key, or if the calling user does not have permission to view the space."
           schema:
             $ref: '#/definitions/confluenceSpaceError'
+  /content/{pageId}:
+    get:
+      summary: "Gets a confluence page content by id."
+      operationId: "getPage"
+      tags:
+        - confluence
+      description: "Returns a piece of Content."
+      security:
+        - crowdAuth: []
+      parameters:
```

Comment:
Missing query parameters for status, version, expand

https://docs.atlassian.com/ConfluenceServer/rest/7.0.4/#api/content-getContentById

Reply:
I'll add them in the next commit